torch/_inductor/fx_passes/split_cat.py

```diff
@@ -122,7 +122,15 @@ def normalize_cat_default(match: Match, *args, **kwargs):
         return
 
     ndim = tensors[0].meta["example_value"].dim()
```

Comment:
This line isn't valid since empty tensors don't need to have the same number of dimensions as the rest of the arguments. Can we take ndim from the output tensor instead?

Reply:
hm, perhaps `cat_node.meta['example_value'].dim()`?  i'll also rewrite the warning to check cat_node and all the tensors for 'example_value'